cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2004,3 +2005,60 @@ action.summary_index = 1
 action.summary_index._name = cyences
 action.summary_index.category = Authentication
 action.summary_index.alert_name = Authentication - Excessive Failed VPN Logins from a Source
+
+
+
+# ===============
+# Linux/Unix
+# ===============
+[Linux - Change in Sudo Access of Local Linux Account]
+disabled = 0
+enableSched = 1
+alert.track = 1
+alert.severity = 3
+alert.suppress = 0
+cron_schedule = 53 * * * *
+counttype = number of events
+quantity = 0
+relation = greater than
+display.general.type = statistics
```

Comment:
Remove these 3 parameters as those are already present below the description. (App inspect failure)